addons/calendar/models/res_partner.py

```diff
@@ -48,7 +48,7 @@ def _compute_meeting(self):
             # Add the events linked to the children of the partner
             for p in self.browse(meetings.keys()):
                 partner = p
-                while partner.parent_id:
+                while partner.sudo().parent_id:
                     partner = partner.parent_id
```

Comment:
The second line is working because partner.parent_id will be in cache (sudoed) after the access through partner.sudo().parent_id. it can be complex to debug or weird to understand as the bugifx rely on a side effect of sudo and cache.

What happens when you open the form ? You have the stat button of the meeting of the partner? The user have access to the meeting once he click on the stat button ?